src/dft_ldos.cpp

```diff
@@ -22,16 +22,31 @@ using namespace std;
 
 namespace meep {
 
-dft_ldos::dft_ldos(double freq_min, double freq_max, int Nfreq) {
-  if (Nfreq <= 1) {
-    omega_min = (freq_min + freq_max) * pi;
-    domega = 0;
-    Nomega = 1;
+// TODO: overload dft_ldos constructor
+dft_ldos::dft_ldos(double *freqs, int Nfreq) {
```

Comment:
If the object makes a copy of `freqs`, then this should be declared as `const double *freqs`.